src/screen.c

```diff
@@ -10397,7 +10397,8 @@ draw_tabline(void)
 	FOR_ALL_TABPAGES(tp)
 	    ++tabcount;
 
-	tabwidth = (Columns - 1 + tabcount / 2) / tabcount;
+	if (tabcount > 0)
+	    tabwidth = (Columns - 1 + tabcount / 2) / tabcount;
```

Comment:
Having `tabcount` equal 0 was odd to me as well, but...

1. `tabcount` is initialised with a zero.
2. If `FOR_ALL_TABPAGES(tp)` on [line 10397](https://github.com/vim/vim/pull/1577/files/9df76ac7b4e620b7bb872de10b8e118e9de530af..14d9d960b8167f872666e344e8daccf0188861be#diff-ceb2e2231c5d45523135ca0cfc492792L10397O) takes a false path, `tabcount` keeps its old value.
3. And it results in division by zero, which is undefined behaviour.